<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Connect</class>
 <widget class="QWidget" name="Form_Connect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resource/resource.qrc">:/Image/background.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Font: 40pt;</string>
   </property>
   <property name="text">
    <string>似乎连接出错了</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>380</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QWidget" name="Group_Set" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>641</width>
     <height>401</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="btn_cancel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_confirm">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_addr">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>地址：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_port">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>端口：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_addr">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_port">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_retry">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>重连</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>lb_title</zorder>
  <zorder>btn_set</zorder>
  <zorder>btn_retry</zorder>
  <zorder>Group_Set</zorder>
 </widget>
 <resources>
  <include location="../Resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
